.circleci/config.yml

```diff
@@ -1050,60 +1050,19 @@ jobs:
         description: Environment overrides
         type: string
         default: ""
-      packages:
-        description: List of packages to test
-        type: string
     machine: true
     resource_class: <<parameters.resource_class>>
     steps:
       - utils/checkout-with-mise
       - attach_workspace:
           at: "."
       - run:
-          name: build op-program-client
-          command: make op-program-client
-          working_directory: op-program
-      - run:
-          name: build op-program-host
-          command: make op-program-host
-          working_directory: op-program
-      - run:
-          name: build cannon
-          command: make cannon
-      - run:
```

Comment:
These steps seem to have gone missing causing the sysgo failures. In the makefile `go-tests-ci` target explicitly says it assumes CI built the dependencies already.

The `cannon-prestate-quick` job is actually building everything we need, but it only persists the prestates to the workspace and we'd need to capture `op-program/bin/op-program`, `op-program/bin/op-program-client`, `cannon/bin/cannon` as well I believe.